main.py

```diff
@@ -105,7 +116,7 @@ def make_mark():
     motor_front_left.run_for_degrees(80, speed=80)
```

Comment:
The number 80 is a somewhat arbitrary number for the speed, but I can call the 80 degrees ANGLE_TO_BOARD, because the 80 degrees moves the marker to the board.